vocabularies/voc4cat/0007004.ttl

```diff
@@ -5,10 +5,11 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 <https://w3id.org/nfdi4cat/voc4cat_0007004>
     a skos:Concept ;
     dcterms:identifier "voc4cat_0007004"^^xsd:token ;
+    skos:broader <https://w3id.org/nfdi4cat/voc4cat_0005007> ;
     skos:definition "The composition of chemical species fed to the inlet of an reactor for a chemical reaction."@en ;
     skos:historyNote "0000-0002-4041-0165 Julia Schumann created the resource"@en ;
     skos:inScheme <https://w3id.org/nfdi4cat/voc4cat_> ;
+    skos:narrower <https://w3id.org/nfdi4cat/voc4cat_0007005> ;
     skos:prefLabel "reaction feed"@en ;
```

Comment:
Shouldn't this be "reactor feed"?

Reply:
right. I added reaction feed as alternate label